settings/authentication.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Single sign-on'
```

Comment:
Can we capitalize and add SSO for Cmd+F-ability?
```suggestion
title: 'Single Sign-On (SSO)'
```

Reply:
updated!